transport/src/main/java/io/camunda/zeebe/transport/stream/impl/messages/StreamTopics.java

```diff
@@ -5,18 +5,17 @@
  * Licensed under the Zeebe Community License 1.1. You may not use this file
  * except in compliance with the Zeebe Community License 1.1.
  */
-package io.camunda.zeebe.protocol.impl.stream;
+package io.camunda.zeebe.transport.stream.impl.messages;
 
-public enum JobStreamTopics {
-  ADD("job-stream-add"),
-  PUSH("job-stream-push"),
-  REMOVE("job-stream-remove"),
-  REMOVE_ALL("job-stream-remove-all"),
-  JOB_AVAILABLE("jobsAvailable");
+public enum StreamTopics {
```

Comment:
:thought_balloon: I wish we had some way of encoding those in the protocol as well :shrug: